### PR TITLE
Run command dr

### PR DESCRIPTION
**Pull request by ankulkarni90**
This PR closes #

The changes in this PR are as follows:

* Includes a cmdlet "Add-GroupToSRMAdmininistrators" to add the group to the "SRM Administrators."
* Includes a cmdlet "Remove-GroupFromSRMAdmininistrators" to remove the group from the "SRM Administrators."
*  Includes a cmdlet "Add-GroupToVRMAdmininistrators" to add the group to the "SRM Administrators."
* Includes a cmdlet "Remove-GroupFromVRMAdmininistrators" to remove the group from the "SRM Administrators."

I have read the [contributor guidelines](CONTRIBUTING.md) and have completed the following:

* [*] **Formatted the code** using VSCode default formatter for PowerShell.
* [*] **Tested the code** end-to-end against an SDDC.
* [*] **Documented the functions** using standard PowerShell markup and applied `AVSAttribute` to newly exported functions.

### CODE REVIEW

**Comment by jupatte** [comment]
Also need to add to the "VRM Administrators" group as well.
That group will only exist if vSphere Replication was installed, but SRM in AVS currently requires vSphere Replication to be useful.

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 1]
```diff
@@ -1027,6 +1027,293 @@ function Remove-GroupFromCloudAdmins {
...
+    $ExternalSources
+    $GroupToAdd
+    $Domain
+
+    try {
+        $ExternalSources = Get-IdentitySource -External -ErrorAction Stop
+    }
+    catch {
+        Write-Error $PSItem.Exception.Message -ErrorAction Continue
+        Write-Error "Unable to get external identity source" -ErrorAction Stop
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1070]: Use `throw` instead, formatting the message with the original error from the excetion.

**Review by jupatte — APPROVED** [review]

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 2]
```diff
@@ -1027,6 +1027,576 @@ function Remove-GroupFromCloudAdmins {
...
+    $SRMAdmins = Get-SsoGroup -Name 'SRM Administrators' -Domain 'vsphere.local'
+    if ($null -eq $SRMAdmins) {
+        throw "Internal Error fetching SRM Administrators group. Contact support"
+    }
+
+    try {
+        Remove-GroupFromSsoGroup -Group $GroupToRemove -TargetGroup $SRMAdmins -ErrorAction Stop
+    }
+    catch {
+        $SRMAdminMembers = Get-SsoGroup -Group $SRMAdmins -ErrorAction Continue
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1305]: nit: ErrorAction is Continue by default, let's not add noise

File: Microsoft.AVS.Management/Microsoft.AVS.Management.psm1 [hunk 3]
```diff
@@ -1027,6 +1027,576 @@ function Remove-GroupFromCloudAdmins {
...
+            }
+            if ($null -ne $GroupFound -and -Not $FoundMatch) {
+                Write-Host "Found $GroupName in $($AD.Name)."
+                $Domain = $AD.Name
+                $GroupToRemove = $GroupFound
+                $FoundMatch = $true
+            }
+            elseif ($null -ne $GroupFound -and $FoundMatch) {
+                Write-Host "Found $GroupName in $($AD.Name) as well."
+                Write-Error "Group $GroupName exists in multiple domains . Please re-run and specify domain" -ErrorAction Stop
```
Comment (et1975) [inline Microsoft.AVS.Management/Microsoft.AVS.Management.psm1:1268]: Use `throw` instead of Write-Error with Stop

**Review by et1975 — CHANGES_REQUESTED** [review]
I've asked Jay to consider moving SRM stuff into a separate internal package we discussed earlier. Let's sync up.

**Comment by et1975** [comment]
As discussed this will go into another repo.

**Pull request closed without merging**